src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const appendRowBtn = document.querySelector('.append-row');
+const removeRowBtn = document.querySelector('.remove-row');
+const appendColBtn = document.querySelector('.append-column');
+const removeColBtn = document.querySelector('.remove-column');
+
+let rowCount = table.rows.length;
+let colCount = table.rows[0].cells.length;
```

Comment:
Ensure that the table has at least one row and one column before accessing `table.rows[0].cells.length`. If the table is empty, this will cause an error. Consider adding a check or initializing the table with a default row and column.